### PR TITLE
Use kubectl to create secret instead of sed

### PR DESCRIPTION
**Pull request by shinyben**
Ha! Rhymes

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/contrib/1348)

### CODE REVIEW

**Comment by eparis** [comment]
This obviously cannot merge until we get the secrets in the utility cluster.  I'll try to do that today or tomorrow.

File: mungegithub/Makefile [hunk 1]
```diff
@@ -80,8 +80,7 @@ endif
 # simple transformation of a github oauth secret file to a kubernetes secret
 secret:
 	@echo $(token64)
```
Comment (eparis) [inline mungegithub/Makefile]: This echo is worthless can you delete it?

**Comment by eparis** [comment]
with that change line 28 is useless (setting token64)

File: mungegithub/Makefile [hunk 2]
```diff
@@ -80,8 +80,7 @@ endif
...
-	sed -e 's|1234567890123456789012345678901234567890123456789012345=|$(token64)|' $(APP)/secret.yaml > $(APP)/local.secret.yaml
-
+	kubectl create secret generic $(APP)-github-token --from-file=token=$(TOKEN)
```
Comment (eparis) [inline mungegithub/Makefile]: This should include `--kubeconfig=$(KUBECONFIG)`

**Comment by apelisse** [comment]
LGTM, @eparis, ready to push the token?

**Comment by apelisse** [comment]
*[a discussion](https://reviewable.kubernetes.io:443/reviews/kubernetes/contrib/1348#-KPPPjXRzXRCwkrUN6xA:-KPPPjXRzXRCwkrUN6xB:b-1o6hv7) (no related file):*
This will probably need rebase. We should have the munger running on contrib by the way @foxish 

---


*Comments from [Reviewable](https://reviewable.kubernetes.io:443/reviews/kubernetes/contrib/1348)*

**Comment by apelisse** [comment]
Reviewed 4 of 5 files at r1, 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.kubernetes.io:443/reviews/kubernetes/contrib/1348)*

**Comment by test-foxish** [comment]
recomputing cla status...

**Comment by grodrigues3** [comment]
[APPROVALNOTIFIER] The Following OWNERS Files Need Approval:

- **mungegithub**

We suggest the following people:
cc @bprashanth 
 You can indicate your approval by writing `/approve` in a comment
 You can cancel your approval by writing `/approve cancel`in a comment

**Comment by apelisse** [comment]
Closing this issue as it's not moving forward. Please re-open if you have any interest in that.

**Comment by apelisse** [comment]
re-read the issue, it's actually something that I would like to see. I'll re-open and get it to merge.

**Comment by apelisse** [comment]
I don't think this is still needed ...

**Pull request closed without merging**